internal/benchrunner/runners/rally/runner.go

```diff
@@ -726,6 +771,138 @@ func (r *runner) runGenerator(destDir string) error {
 	return r.generator.Close()
 }
 
+func countLine(r io.Reader) (uint64, error) {
+
+	var count uint64
+	const lineBreak = '\n'
+
+	buf := make([]byte, bufio.MaxScanTokenSize)
+
+	for {
+		bufferSize, err := r.Read(buf)
+		if err != nil && err != io.EOF {
+			return 0, err
+		}
+
+		var buffPosition int
+		for {
+			i := bytes.IndexByte(buf[buffPosition:], lineBreak)
+			if i == -1 || bufferSize == buffPosition {
+				break
+			}
+			buffPosition += i + 1
+			count++
+		}
+
+		if err == io.EOF {
+			break
+		}
+	}
+
+	return count, nil
```

Comment:
Nit. Wdyt about using an scanner here?
```suggestion
	var count uint64
	scanner := bufio.NewScanner(r)
	for scanner.Scan() {
		count++
	}
	if err := scanner.Err(); err != nil {
		return 0, err
	}
	return count, nil
```

Reply:
I was looking for the most performing way to count newlines from an `io.Reader` and ended up with this: benchmarks were showing magnitude of higher speed

Reply:
Ok, please consider adding a comment mentioning that, to avoid changing this in a future refactor :slightly_smiling_face: